src/views/AnnualReport.vue

```diff
@@ -43,12 +43,24 @@
     />
 
     <!-- Initial Page Load Transition -->
-    <div class="loading-container auto-fade-out">
-      <div class="loading__content">
-        <v-progress-circular color="primary" :size="50" indeterminate></v-progress-circular>
-        <div class="loading-msg">{{loadingMessage}}</div>
+    <v-fade-transition>
+      <div class="loading-container" v-show="showLoadingContainer">
+        <div class="loading__content">
+          <v-progress-circular color="primary" :size="50" indeterminate></v-progress-circular>
+          <div class="loading-msg">{{loadingMessage}}</div>
+        </div>
       </div>
-    </div>
+    </v-fade-transition>
+
+    <!-- Alternate Loading Spinner -->
+    <v-fade-transition>
+      <div class="loading-container grayed-out" v-show="isFetching">
```

Comment:
This is the same grayed out spinner I used when we create a draft IA correction. I'm using it here when the user changes the As Of Date as that requires a new fetch.
(Scott/Tracey may come up with a better spinner style, and if they do then we can fix all the places it's used. But as of now, they're all the same -- consistent -- and not terrible :) )